lib/gpu/context.dart

```diff
@@ -0,0 +1,42 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+part of gpu;
+
+class GpuContextException implements Exception {
```

Comment:
Nit: In general I would stick with the default `Exception` class unless you're adding way more details or you want to extensively document the circumstances under which a developer would `catch GpuContextException` over just a `catch Exception`

Reply:
Removed.